### ISSUE TITLE
Split GUI from Eremetic

### ISSUE THREAD

**Issue by alde** [klarna/eremetic#68]
The GUI should be split into it's own project, to reduce dependencies and simplify the codebase.

**Comment by keis** [comment]
I still would like to have  the option to bundle the gui when deploying. Build time and put into the binary or with the docker image.

**Comment by alde** [comment]
Yes, I agree, but that can still be from a separate project

**Comment by alde** [comment]
Maybe a separate project is unnecessary, but instead make a react frontend that talks to the API

At least that would get rid of the "if accept contains text/html, render template"

**Comment by keis** [comment]
Agreed that we should get rid of the naive search in string bit but having different outputs depending on the accept header is just doing http properly and not hacky nor bad.

**Comment by alde** [comment]
Agreed
